.cruft.json

```diff
@@ -1,34 +1,34 @@
 {
-    "template": "https://github.com/bird-house/cookiecutter-birdhouse",
```

Comment:
I think cruft writes this file.  Maybe exclude it from the formatter as well.  Could leave this here as well, up to you.  We'll see at the next cruft update if cruft will revert this change or able to keep it.

Reply:
I don't exclude it in other projects. So long as the order of entries doesn't get sorted, it should be fine to let the JSON formatter manage whitespace.